support-frontend/assets/helpers/urls/externalLinks.js

```diff
@@ -59,10 +57,10 @@ const memUrls: {
 };
 
 const defaultPromos: PromoCodes = {
-  DigitalPack: getPromoCode('DigitalPack', GBPCountries, 'DXX83X'),
-  Paper: getPromoCode('Paper', GBPCountries, 'GXX83P'),
-  PaperAndDigital: getPromoCode('PaperAndDigital', GBPCountries, 'GXX83X'),
-  GuardianWeekly: getPromoCode('GuardianWeekly', GBPCountries, '10ANNUAL'),
+  DigitalPack: 'DXX83X',
```

Comment:
Are these actually used?

Reply:
It looks like they're only used in the context of building links to the old subscribe platform, which we only do in the (currently commented-out) code for linking to Paper & Digital. I think I'm going to leave this for now but will try and unravel it for the next round of code deleting, because we don't actually seem to need it.